docs/publishing/publishing-nanopublications.md

```diff
@@ -0,0 +1,128 @@
+# Publishing nanopublications
+The `nanopub` library provides an intuitive API that makes publishing nanopublications much easier. 
+The rationale is that you often do not want to worry about the details of composing 
+the RDF that is often the same in each nanopublication. Instead you should focus on the 
+content of your nanopublication: the assertion.
+
+## Prerequisits for publishing
+Before you can publish you should [setup your profile](../setup)
+
+## Quickly publishing nanopublications using `claim`
+You can quickly publish a nanopublicaiton with a single simple statement using the `claim` method:
+```python
+>>> from nanopub import NanopubClient
+
+>>> # Create the client (we use use_test_server=True to point to the test server)
+>>> client = NanopubClient(use_test_server=True)
+
+>>> # Publish a simple statement to the server
+>>> client.claim('All cats are gray')
+Published to http://purl.org/np/RAcTwW_JdTqZJTXA4jUTFUj6ut8OI9AqaPb1IBPsLsy-o
+```
+You can [view](http://test-server.nanopubs.lod.labs.vu.nl/RAcTwW_JdTqZJTXA4jUTFUj6ut8OI9AqaPb1IBPsLsy-o)
+the resulting nanopublication on the nanopub test server. (NB: the purl link actually
```

Comment:
I think it is a good idea that all the docs example code uses the test server by default, as it will help reduce pollution when new users try out the library by just copy-pasting code. But I do think it's a little confusing that the user is immediately faced with the peculiarities of the returned URIs not being the real URIs etc. Perhaps we want a short section on test-server vs no test-server before these code snippets come up?

Reply:
This is a really good point! In the examples I now use `use_test_server=True` but I actually use production URIs so we don't have that confusion there. But this is definitely something users are gonna run into so I've added some separate documentation on this in the 'getting started' part. (I grouped 'What are nanopublications?', 'setup', and 'test server' under 'getting started'.